app/build.gradle

```diff
@@ -79,15 +84,15 @@ android {
     internal {
       dimension "mode"
       applicationIdSuffix "internal"
-      versionName "9.3.0." + getDate()
+      versionName defaultConfig.getVersionName() + "" + getDate()
       versionCode project.VERSION_CODE_VANILLA_INTERNAL.toInteger()
       matchingFallbacks = ['internal', 'prod']
     }
 
     dev {
       dimension "mode"
       applicationIdSuffix "dev"
-      versionName "9.3.0." + getDate()
+      versionName defaultConfig.getVersionName() + "" + getDate()
```

Comment:
is this needed? ``` + ""```

Reply:
defaultConfig.getVersionName() will always return string so it is not necessary - fixed.